### ISSUE TITLE
Generates collisions way too easily

### ISSUE THREAD

**Issue by scottinet** [lovell/highwayhash#2]
I think there is something wrong in the current implementation:

```
$ node -v 
v6.9.2
$ node
> const key = require('crypto').randomBytes(32)
> const hh = require('highwayhash')
{ asBuffer: [Function: asBuffer],
  asString: [Function: asString],
  asHexString: [Function: asHexString],
  asUInt32Low: [Function: asUInt32Low],
  asUInt32High: [Function: asUInt32High] }

hh.asHexString(key, Buffer.from('foobarfoo'))
'b44c325151095a73'

hh.asHexString(key, Buffer.from('foobarfoobar'))
'b44c325151095a73'
```

This should not generate the same hash.
In its current state, I believe this module should not be used.

**Comment by lovell** [comment]
Hello, the latest code in master fixes the problem where only the first 8 bytes were being used - see commit https://github.com/lovell/highwayhash/commit/87648540920026fb7b32ce0f6f2fd72b54ff9b11#diff-8b3531deff59faedeac348ea741c2a24

**Comment by scottinet** [comment]
Is it possible to publish this version on NPM and deprecate the currently available one, as it poses serious security risks?

**Comment by lovell** [comment]
Yes, I agree, will do so shortly.

The commit I referenced also includes the addition of test that would have detected the collision.

**Comment by lovell** [comment]
v2.1.0 now available and v2.0.0 deprecated, thank you.
```sh
$ npm install highwayhash@2.0.0
npm WARN deprecated highwayhash@2.0.0: Please upgrade to v2.1.0 to prevent hash collisions - https://github.com/lovell/highwayhash/issues/2
```
